library/encoding.go

```diff
@@ -52,7 +53,7 @@ func EncodeSymmetric(messageJSON string, symmetricKey string, optionalSigningKey
 		}
 	}
 
-	msg.Version = 1
+	msg.Version = proto.Uint32(1)
```

Comment:
Would be good to define 1 and 0 as constants and use them to avoid magic numbers.